google-cloud-spanner/src/main/java/com/google/cloud/spanner/BackupInfo.java

```diff
@@ -184,6 +215,8 @@ Builder setProto(@Nullable com.google.spanner.admin.database.v1.Backup proto) {
   private final BackupEncryptionConfig encryptionConfig;
   private final EncryptionInfo encryptionInfo;
   private final com.google.spanner.admin.database.v1.Backup proto;
+  private final Timestamp maxExpireTime;
+  private final ProtocolStringList referencingBackup;
```

Comment:
We should not use the protobuf specific types here. Instead of ProtocolStringList, could you use List<String>?